docs/faq/admin.txt

```diff
@@ -109,3 +109,12 @@ Android and iOS.
 
 Depending on feature support, there *may* be minor stylistic differences
 between browsers. These are considered acceptable variations in rendering.
+
+What assistive technologies are supported for using the admin?
+==============================================================
+
+We want the admin to be compatible with wide range of assistive
+technologies, but are aware of many blockers currently. We intend to support
+all latest versions of major assistive technologies, including
+JAWS, NVDA, VoiceOver iOS, TalkBack, Dragon, VoiceOver macOS, Voice Control,
+ZoomText, Windows Contrast Themes, and screen magnifiers.
```

Comment:
Do you think it's worth mentioning so many utilities? :thinking:

Reply:
Not necessarily, I just did this to match what’s done right above for browser support. I’d have felt odd to me to single out specific browsers but not assistive tech.

Reply:
Fine for me, but you have to promise to remove unmaintained options from this list in the future :wink:

Reply:
I promise!

Reply:
I wonder if it's maybe better to mention classes of technologies, "screen readers" rather than listing JAWS, NVDA, etc.

Reply:
@knyghty could you explain why? From my perspective it felt important to match what is done right above for browser support, where I imagine we could also just say "modern browsers" for simplicity but it’s probably clearer for everyone if we mention specific names.

Reply:
Well, fine for me, but you have to promise to remove unmaintained options from this list in the future... and add new ones. Do you think you'll remember?

Reply:
Promise